test/onnx/test_onnx_export.py

```diff
@@ -19,27 +20,66 @@
 sys.path.append(pytorch_test_dir)
 
 
+def export_to_onnx(
+    model: Union[torch.nn.Module, torch.jit.ScriptFunction],
+    input: Tuple[torch.Tensor],
+    custom_ops: Optional[List[Tuple[str, Callable, int]]] = None,
```

Comment:
Should this type for a custom op some `ContextManager` type?

Reply:
Can be anything that implements `__enter__` and `__exit__`